README.md

```diff
@@ -117,6 +147,12 @@ For testing purposes you can create your own LoaderFactory. For ease of use, Str
 
 3rd, 4th and 5th step are same as in previous example.
 
+Since we've added the support for minSdk values of the device you can mock them and customize them when writing tests
```

Comment:
This explanation is strange. Does _setupSdk(20)_ method exists in the library? What is a provider, repository...? You should add a subsection called Writing tests with min SDK and explain this a bit better.